tavla/app/(admin)/edit/[id]/components/MetaSettings/actions.ts

```diff
@@ -105,10 +106,21 @@ export async function moveBoard(
     const access = await hasBoardOwnerAccess(bid)
     if (!access) return redirect('/')
 
-    if (fromOrganization && !(await userCanEditOrganization(fromOrganization)))
-        return redirect('/')
+    if (!personal && !toOrganization)
+        return getFormFeedbackForError('create/organization-missing')
 
-    if (oid && !(await userCanEditOrganization(oid))) return redirect('/')
+    if (!toOrganization)
+        if (fromOrganization) {
+            const canEditFromOrganization =
+                await userCanEditOrganization(fromOrganization)
+            if (!canEditFromOrganization) return redirect('/')
```

Comment:
if if if 
<img src="https://media0.giphy.com/media/gYkga3bZav66I/giphy.gif"/>

Reply:
hahahhaha

Reply:
Den ene ifen skal faktisk ikke være der :not-stonks: